README.md

```diff
@@ -491,6 +501,15 @@ dependencies {
 
 ## Release notes
 
+### 5.7.0-dev
+
+#### Fix:
++ gRPC `retryConfiguration` has been moved from grpc.json to grpc_router.json
++ the whole default gRPC retry interval is about 1 hour
```

Comment:
I believe this is too much for the retry... The suggest we make it 1-5 minutes max. Otherwise, we will cause scripts to wait for a long time to report an error if a component in some kind of crash-look for example...

Reply:
User can sepecify deadlock timeout, also, act component has 10 s default timeout.

Reply:
> act component has 10 s default timeout.

Who said that?) If it is a default value in our template it does not mean that all other act components have the same value.

And what is the `deadlock timeout`? I don't see such a configuration parameter

Reply:
Reduced to 1 minute

Reply:
sorry deadline timeout )